### ISSUE TITLE
Text over 500 letters can be typed into comment

### ISSUE THREAD

**Issue by apple502j** [LLK/scratch-www#2742]
When I typed a lot of letters into comments, I saw "-264 characters left". It shouldn't let users type more than 500.

### Steps to Reproduce
Paste long sentences or type.

### Operating System and Browser
Windows 7/Firefox 65

Reports by community:
mitaku115 https://scratch.mit.edu/discuss/post/3417339/ (env unknown)

**Comment by joker314** [comment]
It's a really useful feature that Twitter also has, and I believe this is by design, for it allows you to rephrase your comment without deleting what you've spent time to write.

**Comment by apple502j** [comment]
At least it should tell the user that the text is over 500 before clicking Send.

**Comment by joker314** [comment]
> At least it should tell the user that the text is over 500 before clicking Send.

I agree. I think negative numbers should show up in red, and the comment box itself should maybe also become red (or orange if red is too scary), and some kind of warning should maybe show up.

But it's not high priority because when you do hit "Send", it doesn't delete your comment, you can still fix it without losing your work

**Comment by thisandagain** [comment]
Thanks @carljbowman. My leaning is to go with the option that does not use a tooltip for now and then add a tooltip should we continue to see issues / confusion.

**Comment by carljbowman** [comment]
@thisandagain - Sounds good. Keep it simple.
